recipes/tesseract/all/conanfile.py

```diff
@@ -204,16 +144,6 @@ def package_info(self):
         elif self.settings.os == "Windows":
             self.cpp_info.components["libtesseract"].system_libs = ["ws2_32"]
 
-        # TODO: to remove in conan v2 once cmake_find_package* & pkg_config generators removed
-        self.cpp_info.names["cmake_find_package"] = "Tesseract"
-        self.cpp_info.names["cmake_find_package_multi"] = "Tesseract"
-        self.cpp_info.components["libtesseract"].names["cmake_find_package"] = "libtesseract"
-        self.cpp_info.components["libtesseract"].names["cmake_find_package_multi"] = "libtesseract"
-        self.cpp_info.components["libtesseract"].build_modules["cmake_find_package"] = [self._module_file_rel_path]
-        self.cpp_info.components["libtesseract"].build_modules["cmake_find_package_multi"] = [self._module_file_rel_path]
-        self.cpp_info.components["libtesseract"].set_property("pkg_config_name", "tesseract")
```

Comment:
Pretty sure `pkg_config_name` is a valid Conan 2 property that is being applied correctly?

Reply:
It's somewhat of a duplicate of the non-component set_property() call. I would suggest getting rid of the single component, that was only added for Conan v1 compatibility, instead.

Reply:
At the moment we will not remove those artificial components, as that could break anyone that references that component explicitly, which can happen in consumers outside of ConanCenter.

We will try to come up with a feature in the client that allows in the future to allow this simplification.